src/main/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1.java

```diff
@@ -281,21 +282,24 @@ public ResponseEntity<Integer> memberAttributeResultsAsync(
     }
 
     /**
-     * Get all the members of an owned grouping through paginated calls.
-     */
+     * Get all the members of an owned grouping through paginated calls. This should be done through the UI using await.
+     * Currently, the UI is not using this function to hydrate grouping members, this implementation of getMembers is
+     * much faster than the current getMembers in use and should be used in the future ether when GROUPINGS-304 is
+     * completed on the UI or when the UI is migrated to the react framework.
```

Comment:
Could you remove the sentences starting from "This should  be done through the UI using await." The comments are outdated, thanks.